sdbuild/ubuntu/bionic/aarch64/multistrap.config

```diff
@@ -8,6 +8,7 @@ addimportant=true
 [Bionic]
 source=http://ports.ubuntu.com/ubuntu-ports
 suite=bionic universe
+keyring=ubuntu-keyring
```

Comment:
Maybe this could be added to `sdbuild/ubuntu/bionic/arm/multistrap.config`.

I was encountering issues with GPG and not verifiable keys (3B4FE6ACC0B21F32) and your fix solved it.
However, I am working with a customized version of the Pynq-Z2, so I had to apply your changes to arm platforms.

(Working with Virtual Box, Ubuntu 18.04.3 LTS)
--
Btw, thanks for your help.

Reply:
Updated to build for arm boards too. Thx for your feedback.